.travis.yml

```diff
@@ -14,5 +14,8 @@ android:
     - extra-android-m2repository
     - addon-google_apis-google-19
 
-    - sys-img-armeabi-v7a-android-22
-    - sys-img-x86-android-17
\ No newline at end of file
+    - sys-img-armeabi-v7a-android-24
+    - sys-img-x86-android-17
+
+script:
+    - ./gradlew test
```

Comment:
I think it's best to do travis stuff on a separate branch and then get rid of it, so nobody has to see all the inevitable "Fixes travis" commits.